ui/articles-aem-renderer/src/main/java/org/cru/godtools/articles/aem/service/support/ArticleParser.java

```diff
@@ -172,6 +190,21 @@ private static long getDateLongFromJsonString(String dateString) {
             return new Date().getTime();
         }
     }
+
+    /**
+     * Using "https://stage.cru.org/content/experience-fragments/questions_about_god/english.9999.json"
+     * as an example url this method will remove ".999.json" from the url leaving the base url.
+     * @param url
+     * @return
+     */
+    private static String getArticleBaseUrl(String url) {
+        if (url.endsWith(".json")) {
+            url = url.substring(0, url.lastIndexOf("."));
```

Comment:
I don't think this will actually be necessary since we should never be passing in a url that has `.###.json`.

But if we did need to sanitize the url, this logic is brittle. If the url is just `https://stage.cru.org/english.json`, This will  throw a `StringIndexOutOfBoundsException`. How you would probably want to handle this is through a regular expression.